cmake/modules/cpp-httplib.cmake

```diff
@@ -18,6 +18,10 @@ option(USE_BUNDLED_CPPHTTPLIB "Enable building of the bundled cpp-httplib" ${USE
 if(USE_BUNDLED_CPPHTTPLIB)
 	set(HTTPLIB_USE_BROTLI_IF_AVAILABLE OFF)
 	set(HTTPLIB_REQUIRE_BROTLI OFF)
+	set(HTTPLIB_USE_ZLIB_IF_AVAILABLE OFF)
+	set(HTTPLIB_REQUIRE_ZLIB OFF)
+	set(HTTPLIB_USE_OPENSSL_IF_AVAILABLE OFF)
+	set(HTTPLIB_REQUIRE_OPENSSL OFF)
```

Comment:
cpp-httplib strikes again! I don't think we use compression or encryption in `webserver.cpp`

Reply:
For compression, you are likely correct. The versions endpoint is `application/json` and the metrics endpoint is currently `text/plain; version=0.0.4` using the prometheus exposition format.

However, for encryption, there's the SSL Enabled option.

Reply:
Makes sense, I have re-enabled OpenSSL. Fortunately it does recognize the static OpenSSL build. Thanks a lot @sgaist !